### PR TITLE
add r-sctransform

### PR DESCRIPTION
**Pull request by dpryan79**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by dpryan79** [comment]
@conda-forge/help-r This is ready for review. It's needed for the new release of Seurat over on bioconda.

File: recipes/r-sctransform/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    - r-reshape2
+
+test:
+  commands:
+    - $R -e "library('sctransform')"           # [not win]
+    - "\"%R%\" -e \"library('sctransform')\""  # [win]
+
+about:
+  home: https://github.com/ChristophH/sctransform
+  license: GPL-3
```
Comment (bgruening) [inline recipes/r-sctransform/meta.yaml]: the license_file is missing here, but we need it because its GPL :(

Comment (dpryan79) [reply recipes/r-sctransform/meta.yaml]: Haven't we always just ignored that? Anyway I'll add it.

Comment (dpryan79) [reply recipes/r-sctransform/meta.yaml]: I think it was removed from the skeleton since so few packages actually ship the license. Anyway, it's added now.

Comment (bgruening) [reply recipes/r-sctransform/meta.yaml]: Look at the checklist above ;)

**Pull request merged**